pkg/terraform/terraform.go

```diff
@@ -272,8 +274,32 @@ func (t *Terraform) Destroy(cluster interfaces.Cluster) error {
 }
 
 func (t *Terraform) Shell(cluster interfaces.Cluster) error {
-	// TODO: needs to be implemented
-	return fmt.Errorf("Shell unimplemented")
+
+	if err := t.terraformWrapper(cluster, "", nil); err != nil {
+		return err
+	}
+
+	dir := t.codePath(cluster)
+	envVars, err := t.envVars(cluster)
```

Comment:
It would be good to get all the environment variables and overwrite only the ones from t.envVars